arduino.js

```diff
@@ -85,13 +92,13 @@ function initTasks(gulp, options) {
   });
 
   gulp.task('build', 'Builds sample code', function (cb) {
-    all.writeConfigH();
+    all.writeConfigH(true);
     all.localExecCmd(getArduinoCommand() + ' --verify --board ' +
       boardDescriptor + ' ' + process.cwd() + '/app/app.ino --verbose-build', args.verbose, cb);
   });
 
   gulp.task('deploy', 'Deploys binary to the device', function (cb) {
-    all.writeConfigH();
+    all.writeConfigH(true);
     var port = null;
```

Comment:
I think this function needs to be called only once. How about only calling it in deploy task. 

For raspberry pi C, we removed the build task.